src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -558,6 +553,22 @@ class DocBaseViewer extends BaseViewer {
             });
     }
 
+    /**
+     * Get pdf.js viewer.
+     *
+     * @protected
+     * @override
+     * @return {PDFJS.PDFViewer} PDF viewer type
+     */
+    getPdfViewer() {
```

Comment:
since we're actually instantiating the viewer here as opposed to getting it, can you rename this to `initPdfViewer`?